vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -985,15 +986,24 @@ public User getOrCreateUser(String email, String defaultAccountType)
                 }
             }
 
-            user = getNewUser(email, firstName, lastName);
+            if(egi_affiliation != null){
```

Comment:
Put this business into EgiController as this is specific to egi

Reply:
Done